app/assets/javascripts/comments.js

```diff
@@ -200,6 +200,19 @@ $(() => {
     const splat = content.split(' ');
     const caretPos = $tgt[0].selectionStart;
     const [currentWord, posInWord] = currentCaretSequence(splat, caretPos);
+    const postId = $tgt.data('post');
+    const characterCount = document.getElementById("character_count_"+postId);
+    const createThreadButton = document.getElementById("create_thread_button_"+postId);
+
+    if (caretPos<15 || caretPos>500){
+        characterCount.style.color = "#ff0000";
+        createThreadButton.disabled = true;
+    }else if(caretPos>375){
+        characterCount.style.color = "#d7af02";
+    }else{
```

Comment:
Likewise here

Reply:
This applies to all instances of this formatting. Try to follow the style of the existing JS in the project - new lines before every new if/else if/else block.